### PR TITLE
Resolved #669 on 0.26-upgrade

### LINKED ISSUE #669: "Make a request" button on 2 lines in 0.26
image

### PR DESCRIPTION
**Pull request by Keerti-Gautam**
As a genuine concern pointed out by @benrfairless in #684 , I am creating a new PR into the branch `0.26-upgrade` so the changes can be merged in a single hit. 
This change of the `max-width: 11em` which puts back "Make a Request" in a single line, works well with older as well as newer versions of Right To Know.

### CODE REVIEW

File: assets/stylesheets/responsive/custom.scss [hunk 1]
```diff
@@ -445,7 +445,9 @@ color: $righttoknow-heading-color;
 }
 
 .make-request-quick-button {
-  margin-top: 1em;
+    max-width: 11em;
```
Comment (equivalentideas) [inline assets/stylesheets/responsive/custom.scss]: This looks like it uses different indentation and an extra line of whitespace compared to the rest of this file. Could you please fix add a commit to correct that @Keerti-Gautam

**Review by equivalentideas — CHANGES_REQUESTED** [review]
Thanks for this contribution @Keerti-Gautam :) Fixing things like this will make the upgrade much quicker 💯 

I've left a comment requesting changes of the style of your css to match the rest of the file. It's important files are consistent, and when making a PR to an open source project make sure you're change is in line with the projects code style.

Also in the future it would be great to have a more descriptive commit message and Pull request. Here's a couple of great guides I often refer to myself :) :

https://github.com/blog/1943-how-to-write-the-perfect-pull-request

https://robots.thoughtbot.com/5-useful-tips-for-a-better-commit-message

**Comment by Keerti-Gautam** [comment]
Hi Luke,
I had added a thorough description of the changes on the last PR I made #684, which required a version check as correctly pointed out by @benrfairless so I created another PR here. If it was missed out, I will just add the content here for you to review! 
Thanks a lot for your suggestions! Will give them a read today! :smile: 
...
"Upon upgrading the version of alaveteli, this button was somehow broken and 'Make a Request' was coming on two lines. It was because of the addition of a class in alaveteli which is `body_listing__request-button`, which made me realize that this was a design altercation in alaveteli and not in RightToKnow.
But I searched a little more and came to find out that [WhatDoTheyKnow](https://www.whatdotheyknow.com/) is another FOI website used in UK and it uses the current alaveteli version but still, 'Make a Request' comes in a single line.
Hence it was clear that some changes had to be made in RightToKnow to fix the issue.

*Changes Made:*
The `max-width` of the button was hence changed from its inherited 10em to 11em.
Screenshot attached."

screenshot

**Review by equivalentideas — APPROVED** [review]

**Comment by equivalentideas** [comment]
Very nice @Keerti-Gautam :) This does the job!

**Pull request merged**